documentation/IDTA-01001/modules/ROOT/pages/Spec/IDTA-01001_Metamodel_Constraints.adoc

```diff
@@ -70,4 +70,27 @@ Note: The semanticId of a SpecificAssetId with the predefined name "gloablAssetI
 
 {aasd130}
 
+An attribute with data type "string" shall consist of these characters only:
```

Comment:
```suggestion
We have to restrict an attribute with data type "string" to the characters which are representable in XML.
Otherwise, strings in other formats such as JSON could not be converted to XML.

The character set of XML includes (given as numerical code points and/or ranges in Unicode):
* 0x09: ASCII horizontal tab,
* 0x0A: ASCII linefeed (newline),
* 0x0D: ASCII carriage return.
* 0x20: ASCII space,
* 0x20 - 0xD7FF: all the characters above space in the Basic Multilingual Plane, and
* 0x00010000-0x0010FFFF: all the characters beyond the Basic Multilingual Plane (*e.g.*, emoticons).
```